File: Java/ejer02/src/ejer02/Ejer02.java

package ejer02;


public class Ejer02 {

    /**
Crea una clase Empleado con nombre y sueldo
a)
Agrega un método para calcular el sueldo anual
b)
Agrega un método para aplicar un aumento del 10%
c)
Crea dos empleados y muestra sus sueldos antes y después del aumento
     */
    public static void main(String[] args) {
        // TODO code application logic here
        
        Empleado e2=new Empleado("maria", 3467.4);
        Empleado e3=new Empleado("Marturian", 40.4);
        System.out.println(e2.getSueldo());
        System.out.println(e3.getSueldo());
        System.out.println("despues del aumento");
        e2.aplicarAumento();
        e3.aplicarAumento();
        System.out.println(e2.getSueldo());
        System.out.println(e3.getSueldo());
        
        
        System.out.println(e2.toString());
        e2.aplicarAumento(); //llamamos a la funcion para aplicar aumento
        System.out.println(e2.toString());
        
        //double sueldoAnual = e1.sueldoAnual();
        //System.out.println("El salario anual del empleado es:" );
        //System.out.println(sueldoAnual);
    }
    
}

File: Python/ejer02.py
class Empleado:
    def __init__(self,nombre,sueldo):
        self.__nombre = nombre
        self.sueldo = sueldo
    def __str__(self):
        return f'Nombre: {self.__nombre} Salario: {self.sueldo}'
    
    def sueldoAnual(self):
        sueldoAnual = self.sueldo*12
        return sueldoAnual
    def aumento(self):
        self.sueldo = self.sueldo*(1.10)
    def getNombre(self):
        return self.__nombre
        
class main():
    emp01 = Empleado("jose", 1800.50)
    emp02 = Empleado("pedrito", 7826.50)
    print("Empleado 1:")
    print(emp01)
    print("Empleado 2:")
    print(emp02)
    print(f'Sueldo anual empelao 1:{emp01.sueldoAnual()}')
    print(f'Sueldo anual empelao 2:{emp02.sueldoAnual()}')
    emp01.aumento()
    emp02.aumento()
    print("empleado 1 despues del aumento")
    print(emp01)
    print("empleado 2 despues del aumento")
    print(emp02)
    print(emp01.getNombre())